test/integration/provider/gcp.go

```diff
@@ -53,6 +53,23 @@ func getMachines(machineClass *v1alpha1.MachineClass, secretData map[string][]by
 	return machines, nil
 }
 
+func cleanUpOrphanResources(instanceIds []string, volumeIds []string, ctx context.Context, svc *compute.Service, project, zone string) (delErrInstanceId []string, delErrVolumeIds []string) {
+
+	for _, instanceId := range instanceIds {
+		if err := terminateInstance(ctx, svc, project, zone, instanceId); err != nil {
+			delErrInstanceId = append(delErrInstanceId, instanceId)
+		}
+	}
+
+	for _, volumeId := range volumeIds {
+		if err := deleteVolume(ctx, svc, project, zone, volumeId); err != nil {
+			delErrVolumeIds = append(delErrVolumeIds, volumeId)
+		}
+	}
+
+	return
```

Comment:
why not return `delErrInstanceId` and `delErrVolumeId` ?

Reply:
named return parameters are returned like this only.